lib/db.js

```diff
@@ -36,7 +37,10 @@ const _task = (obj, opts, cb) => {
   })
 }
 
-const getSchema = () => 'public'
+const getSchema = () => {
+  const store = asyncLocalStorage.getStore() ?? defaultStore()
```

Comment:
I don't think we should go to defaultStore here. If the storage isn't set at this point i think the query should fail fast on lamassu so we can spot it before we get silent issues in pazuz